backend/src/battles/const/battles.const.ts

```diff
@@ -0,0 +1,30 @@
+export enum BATTLE_LANGUAGE {
+  TS = 'TS',
+  JS = 'JS',
+  PYTHON = 'PYTHON',
+}
```

Comment:
```suggestion
export const BATTLE_LANGUAGE = {
  TS: 'TS',
  JS: 'JS',
  PYTHON: 'PYTHON',
} as const;

export type BattleLanguage =
  typeof BATTLE_LANGUAGE[keyof typeof BATTLE_LANGUAGE];
```

`enum`은 TS에서 안티 패턴이라 런타임에서 좋지 않다고 알고 있는데, 이렇게 타입으로 상수처리하는 건 어떠신가요?

Reply:
좋습니다! 반영했습니다!